### PR TITLE
Code example tweaks

### PR DESCRIPTION
**Pull request by Paul-Hebert**

### CODE REVIEW

File: src/group/example-a.njk [hunk 1]
```diff
@@ -33,6 +33,7 @@ css:
   withBrowser(async ({
     screen, utils, page, user
   }) => {
+
```
Comment (Paul-Hebert) [inline src/group/example-a.njk:36]: This makes the spacing consistent in this code block (blank line before each comment)

File: src/group/example-a.njk [hunk 2]
```diff
@@ -103,15 +104,15 @@ css:
...
-  test(
-    'test name',
-    withBrowser(async () => {
-      // Your test code here
-    }),
+test(
+  'test name',
+  withBrowser(async () => {
+    // Your test code here
+  }),
```
Comment (Paul-Hebert) [inline src/group/example-a.njk:115]: This fixes some odd indentation

File: src/group/example-a.njk [hunk 3]
```diff
@@ -125,20 +126,20 @@ css:
         <div class="offset">
           <div class="code-wrapper">
         <pre class="code-block"><code class="language-js">// For JS frameworks you can run arbitrary JavaScript
+// (./app could be a .js, .jsx .ts, or .tsx file)
```
Comment (Paul-Hebert) [inline src/group/example-a.njk:129]: I thought this might read better moved outside the function

File: src/group/example-a.njk [hunk 4]
```diff
@@ -125,20 +126,20 @@ css:
...
-  // ./app could be a .js, .jsx .ts, or .tsx file
   import { App } from './app'
   import { render } from 'react-dom'
 
   render(<App />, document.body)
-  `);
-  // You can also insert HTML
-  (This works well with templating languages)
-  await utils.injectHTML('<h1>Find Hikes</h1>');
+`);
```
Comment (Paul-Hebert) [inline src/group/example-a.njk:135]: I had this indentation wrong in my content doc

File: src/group/example-a.njk [hunk 5]
```diff
@@ -125,20 +126,20 @@ css:
...
+
+// You can also insert HTML
```
Comment (Paul-Hebert) [inline src/group/example-a.njk:137]: Make blank lines consistent

File: src/group/example-a.njk [hunk 6]
```diff
@@ -125,20 +126,20 @@ css:
...
+// (This works well with templating languages)
+await utils.injectHTML('<h1>Find Hikes</h1>');
 
-  // Tests can also be run by navigating
-  to URLs within your app:
-  await page.goto('http://localhost:3000/hikes');</code></pre>
+// Tests can also be run by navigating to URLs within your app:
+await page.goto('http://localhost:3000/hikes');</code></pre>
```
Comment (Paul-Hebert) [inline src/group/example-a.njk:142]: Fix indentation of this section

File: src/group/example-a.njk [hunk 7]
```diff
@@ -150,19 +151,14 @@ await utils.runJS(`
...
-  )
-);</code></pre>
+            <pre class="code-block"><code class="language-js">const locationInput = await screen.getByRole( 'textbox', {
+  name: 'Pick a Location'
+});
+const submitButton = await screen.getByRole(‘button’, {
+  name: 'Find Hikes'
+});
+await user.type(locationInput, 'Portland Oregon');
+await user.click(submitButton);</code></pre>
```
Comment (Paul-Hebert) [inline src/group/example-a.njk:161]: Caleb asked to reorganize this code a bit

File: src/group/example-a.njk [hunk 8]
```diff
@@ -174,7 +170,8 @@ await user.click(
         <p>Use Jest-Dom’s intuitive syntax to confirm your interface responds correctly to user actions:</p>
         <div class="offset">
           <div class="code-wrapper">
-            <pre class="code-block"><code class="language-js">await expect(hikeResults).toBeVisible();</code></pre>
+            <pre class="code-block"><code class="language-js">await expect(submitButton).toHaveFocus();
+await expect(hikeResults).toBeVisible();</code></pre>
```
Comment (Paul-Hebert) [inline src/group/example-a.njk:174]: Caleb asked to add a second assertion

**Review by AriannaChau — APPROVED** [review]

**Pull request merged**